src/bitmessageqt/migrationwizard.py

```diff
@@ -1,26 +1,39 @@
 #!/usr/bin/env python2.7
```

Comment:
Unused, unfinished and obsolete. Remove whole file.

Reply:
+1 for ruthlessness :)